src/main/java/pl/med/clinic/dto/VisitDtoRequest.java

```diff
@@ -0,0 +1,19 @@
+package pl.med.clinic.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import pl.med.clinic.entity.Paid;
+import pl.med.clinic.entity.Payment;
+
+import java.time.LocalDateTime;
+
+@Getter
+@AllArgsConstructor
+public class VisitDtoRequest {
+
+    private LocalDateTime visitDateTime;
```

Comment:
zdefiniuj format daty i czasu, sugeruję taki sam jak wcześniej, 
a może znajdziesz sposób na globalne ustawienie formatu dla całej aplikacji ??
`    @JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss")`

Reply:
usunełam z pattern z visitDto i ustawiłam w application properties, zakładam, że teraz nie muszę w każdym tego dodawać, ale jak dodam nadpisywany jest pattern globalny?

Reply:
spring.jackson.date-format=yyyy-MM-dd HH:mm:ss

Reply:
ja po teście stwierdzam że nie działa a z wcześniejszą adnotacją działało

Reply:
data-godzina wizyty: 
zmianiłam format z localdate na date i wtedy globalnie formatter działa (plus time zone)
ale w swaggerze widze to dobrze w respondzie, a w requscie tamplate musze zmieniac - do przegadania czy tu powinnam stworzyć formatter dla LocalDate

birth date:
zwykle daty zostawilam LocalDate bez formatowania bo tam działa dobrze